en/key_concepts.rst

```diff
@@ -3,10 +3,13 @@ Key concepts
 
 This document will explain the most important concepts to start with Open Orchestra.
 
-Nodes
+Pages
 -----
 
-Nodes are mainly pages that the visitors can see on the website, they are containers for zones and blocks.
+Nodes
+~~~~~
+
+In the Open Orchestra project, a page is identified as a node. They are containers for zones and blocks.
```

Comment:
I think project is not adequate, I think "solution" or "cms" should be prefered to talk about the product.